src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/ThorchainSaversDeposit.tsx

```diff
@@ -175,6 +175,15 @@ export const ThorchainSaversDeposit: React.FC<YearnDepositProps> = ({
           />
         ),
       },
+      ...(assetId === usdtAssetId
```

Comment:
Ensures we only display this guy for mainnet USDT... which on a tangent, we should also apply to the "Consolidate" step for UTXOs onl 
<img width="517" alt="image" src="https://github.com/user-attachments/assets/bf2e3744-d398-4e78-b487-1119edbd4c92">